files/templates/scrapeconfigs/additional-scrape-configs.template.yaml

```diff
@@ -1206,3 +1206,31 @@
   - replacement: prometheus
     target_label: app
 [[ include "_labelingschema" . | indent 2 ]]
+[[- if eq .ClusterType "management_cluster" ]]
+# cilium agent and cilium operator
+- job_name: [[ .ClusterID ]]-prometheus/cilium-[[ .ClusterID ]]/0
```

Comment:
If this is tested?

Reply:
it's been running on gremlin, yeah

Reply:
![up](https://user-images.githubusercontent.com/868430/195646895-bd64f062-854a-4e9c-9d2e-0e3f1d7dbb3c.png)